.ci/generate_test_report_lib.py

```diff
@@ -122,6 +122,17 @@ def plural(num_tests):
             ]
         )
 
+    if failures or return_code != 0:
+        report.extend(
+            [
+                "",
+                "If these failures are unrelated to your changes (e.g., tests are "
```

Comment:
e.g. -> "for example", which is a bit easier to read.

This is markdown so I assume that the github link will be made clickable automatically.

I think you should highlight the infrastructure word somehow, to make it extra clear that there is literally a label "infrastructure". You could do that with `""` or put it in backticks so it gets the plaintext formatting.

Reply:
Some folks won't have permissions to add labels but mentioning that here probably causes more confusion. We can hope people will open the issue anyway, and it'll be pretty clear it's to do with infrastructure regardless.

Reply:
Updated to `s/e.g.,/for example` and `s/infrastructure label/<back tick>infrastructure<back tick> label`.

Yeah, the permissions might be an interesting challenge. If it becomes a big issue we can modify the message in the future. If the issue volume is pretty high we might also want to create a `premerge` label at some point, but I think we can do that when lumping everything in with `infrastructure` becomes a problem.